### PR TITLE
appdata: Add release information

### LINKED ISSUE #41: Appdata is missing release version
```xml
<releases>
  <release version="1.2.3" date="2018-02-04"/>
</releases>
```

Or such. Even better you can include lists of new features but that is understandably a lot more work that ideally upstream does.

### PR DESCRIPTION
**Pull request by RobLoach**
Fixes #41

### CODE REVIEW

File: org.libretro.RetroArch.appdata.xml [hunk 1]
```diff
@@ -17,4 +17,139 @@
   <url type="homepage">http://retroarch.com</url>
   <project_license>GPL-3.0</project_license>
   <metadata_license>CC0-1.0</metadata_license>
+  <releases>
+    <release version="1.7.1" date="2018-02-19" urgency="medium">
```
Comment (TingPing) [inline org.libretro.RetroArch.appdata.xml]: `"medium"` is the default btw, no harm eitherway though.

File: org.libretro.RetroArch.appdata.xml [hunk 2]
```diff
@@ -17,4 +17,139 @@
...
+          <_li>SCANNER: Fix crash from Windows-incompatible format string.</_li>
+          <_li>VITA: Improve packaging, installation times.</_li>
+          <_li>WIIU: Disabled the controller patcher for now since it was the source of many stability issues.</_li>
+          <_li>VULKAN: Various stability fixes for WSI.</_li>
+          <_li>WINDOWS: Add MSVC 2017 solution.</_li>
+          <_li>WINDOWS: Get rid of the empty console window in MSVC 2010 builds.</_li>
+          <_li>WINDOWS: Raw input driver now supports new lightgun code.</_li>
+          <_li>WINDOWS: Use configured OSD/text message color on GDI driver.</_li>
+          <_li>WINDOWS/XINPUT: Populate XInput VID/PID from DInput so autoconfig doesn't rely solely on joypad names</_li>
+          <_li>WINDOWS/XINPUT: Fix crash that occurs in some situations with Steam running and a Steam Controller plugged in.</_li>
```
Comment (TingPing) [inline org.libretro.RetroArch.appdata.xml]: These shouldn't have leading `_`'s

File: org.libretro.RetroArch.appdata.xml [hunk 3]
```diff
@@ -15,139 +15,139 @@
     </p>
   </description>
   <url type="homepage">http://retroarch.com</url>
-  <project_license>GPL-3.0</project_license>
-  <metadata_license>CC0-1.0</metadata_license>
+  <projectlicense>GPL-3.0</projectlicense>
+  <metadatalicense>CC0-1.0</metadatalicense>
```
Comment (TingPing) [inline org.libretro.RetroArch.appdata.xml]: Your find and replace was too broad :P

Comment (RobLoach) [reply org.libretro.RetroArch.appdata.xml]: LOL! Oops.

**Pull request merged**